Changelog.md

```diff
@@ -1,3 +1,10 @@
+## 4.3.1 - Unreleased
+
+### Fixed
+
+-   The **Enable MCUboot** toggle reflects the MCUboot trait of the connected
+    device.
```

Comment:
I wouldn't say this is a bug fix, but rather just a change in the behavior. From my perspective this toggle had a different goal and didn't have to reflect the state of the device (might be I'm wrong).

```suggestion
### Changed

-   The **Enable MCUboot** toggle reflects the MCUboot trait of the connected
    device.
```

Reply:
I believe it should be fixed as this was the previous UI and this can be considered as a regression.

Reply:
Not my piece of cake, but Kyle's argument seems valid to me at this point of this feature's life cycle.